### DISCUSSION TITLE
How to send events to Server Side Tag Manager without using GA4?

### DISCUSSION THREAD

**Discussion by anooptu** [elbwalker/walkerOS#459] [General]
I want to send events to SSTM and not planning to use GA4. Can I do this using WalkerOS?

**Comment by alexanderkirtzel** [comment]
To send events from the frontend client side, you can use the [API destination](https://www.elbwalker.com/docs/destinations/web/api), which sends original walkerOS events.

```ts
import { destinationWebAPI } from '@elbwalker/destination-web-api';

elb('walker destination', destinationWebAPI, {
  custom: {
    url: 'YOUR_SGTM_URL',
    transport: 'beacon',
  },
});
```
See [how to use](https://www.elbwalker.com/docs/destinations/web/api#how-to-use) if you want to load it via script.

Or the `etag` destination, which is currently under development. It works almost like the original `gtag` library and creates requests only based on walkerOS. There are no external Google dependencies required.

You can also use `etag` with node to send events from a backend server-side. Or send original requests via [sendNode](https://www.elbwalker.com/docs/utils/helper#sendnode)
